### PR TITLE
fixes for json format changes

### PR DESCRIPTION
**Pull request by FodT**

### CODE REVIEW

File: terminalone/jsonparser.py [hunk 1]
```diff
@@ -183,3 +189,10 @@ def dictify_access_flag(flag):
             if 'id' == key or key.endswith('_id'):
                 output[key] = int(output[key])
         return output
+
+    @classmethod
+    def get_first(cls, iterable, default=None):
+        if iterable:
+            for item in iterable:
+                return item
+        return default
```
Comment (pswaminathan) [inline terminalone/jsonparser.py]: What is the difference between this and `next(iter(iterable), default=default)`?

File: terminalone/jsonparser.py [hunk 2]
```diff
@@ -50,10 +50,10 @@ def __init__(self, body):
...
             self.entities = map(self.process_entity, data)
 
-        elif parsed_data.get('include') is not None or \
-                parsed_data.get('exclude') is not None or \
-                parsed_data.get('enabled') is not None:
-            self._parse_target_dimensions(parsed_data)
+        elif data.get('include') is not None or \
+                data.get('exclude') is not None or \
+                data.get('enabled') is not None:
+            self._parse_target_dimensions(data)
```
Comment (pswaminathan) [inline terminalone/jsonparser.py]: Why were we using `parsed_data` before? Seems like we should have been using `data` all along.

Comment (FodT) [inline terminalone/jsonparser.py]: I'm not actually sure. Either there was a time when the API wasn't returning target_dimensions wrapped alongside meta objects, or this just, never worked. Either way, it's fixed now!

Comment (pswaminathan) [inline terminalone/jsonparser.py]: Ohhh I do remember this now. The API wasn't returning a meta object for target_dimensions.

**Comment by pswaminathan** [comment]
:+1:

**Pull request merged**